app/config/routing_test.yml

```diff
@@ -8,7 +8,6 @@ surfnet_stepup_gateway_behat_service_provider_acs:
 
 surfnet_stepup_gateway_behat_identity_provider_acs:
     path:     /test/authentication/sso
-    methods:  [GET]
```

Comment:
Why not use 'GET' and 'POST' as defined in the prod routes?

Reply:
Yeah, good point was just yanking stuff out, to get it to work to be honest. I will fix this right away.